common/schema/definition.go

```diff
@@ -376,6 +378,11 @@ END`,
 					`dictGetOrDefault('icmp', 'name', tuple(Proto, ICMPv6Type, ICMPv6Code), ` +
 					`concat(toString(ICMPv6Type), '/', toString(ICMPv6Code))), '')`,
 			},
+			{
+				Key:             ColumnNextHop,
+				ClickHouseType:  "IPv6",
```

Comment:
I think there are not many nexthops available. You could go with `LowCardinality(IPv6)` instead.